src/elm/Help/Commands.elm

```diff
@@ -799,6 +838,9 @@ resourceLoading args =
         Pages.RepositoryBuilds _ _ _ _ _ ->
             args.builds.loading
 
+        Pages.RepositoryDeployments _ _ _ _ ->
+            args.builds.loading
```

Comment:
Same here, what should this be?

Reply:
sorry for the late reply, i missed this comment!
see my explanation in the org page review:
https://github.com/go-vela/ui/pull/424#discussion_r680065056